vassal-app/src/main/java/VASSAL/build/module/turn/TurnLevel.java

```diff
@@ -406,6 +407,9 @@ public ComponentI18nData getI18nData() {
    */
   @Override
   public List<String> getPropertyList() {
-    return List.of(propertyName);
+    if (propertyName != null) {
+      return List.of(propertyName);
+    }
+    return Collections.emptyList();
```

Comment:
`List.of()` is the same thing and you don't need to import `Collections`, then.

Now more concise and less verbose: :)
```
return propertyName != null ? List.of(propertyName) : List.of();
```

Reply:
done